ChegeBryan/politico
# #163545907 Political parties overview (admin)

#### What does this PR do?

Creates an admin dashboard political parties overview.

#### Description of Task to be completed?

- [x] Create a navbar
- [x] Create a side navigation bar
- [x] Create a table with political parties details
- [x] Create a modal to add political parties
- [ ] Make the page responsive

#### How should this be manually tested?
clone this at `https://github.com/ChegeBryan/politico.git` and check out to `ft-admin-dashboard-163545907` branch. Navigate to the `UI` directory,  Open the file `admin_dashboard.html` on your favourite browser.

#### What are the relevant pivotal tracker stories?
[#163545907](https://www.pivotaltracker.com/story/show/163545907)

#### Screenshots.
##### On desktop
![screencapturei-admin_dashboard-html-2019-01-31-11_01_14](https://user-images.githubusercontent.com/40359451/52040292-faac1c80-2547-11e9-854f-7dfaaf766e68.png)

##### form to create party
![screencaptur--ui-admin_dashboard-html-2019-01-31-15_50_30](https://user-images.githubusercontent.com/40359451/52055309-17f4e100-2570-11e9-8472-b5c360e353f2.png)
